lib/graphql/static_validation/rules/variables_are_used_and_defined_error.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+module GraphQL
+  module StaticValidation
+    class VariablesAreUsedAndDefinedError < Message
+      attr_reader :variable_name
+      attr_reader :violation
+
+      VIOLATIONS = {
+        :VARIABLE_NOT_USED      => "variableNotUsed",
+        :VARIABLE_NOT_DECLARED  => "variableNotDeclared",
```

Comment:
I'm sorry this is a nitpick, but what do you think about:

```ruby
        :VARIABLE_NOT_DEFINED  => "variableNotDefined",
```

The only reason I'm suggesting this is because the class name uses the word "defined" and [so does the GraphQL spec](https://facebook.github.io/graphql/June2018/#sec-All-Variable-Uses-Defined).

Reply:
Sure thing - I'll sort this 👍